crates/viewer/re_viewer/Cargo.toml

```diff
@@ -45,6 +45,8 @@ grpc = ["re_data_source/grpc", "dep:re_grpc_client"]
 
 
 [dependencies]
+# XXX TODO FIX THIS IMPORT:
+color-hex = { version = "*", default-features = false }
```

Comment:
I would like to remove this in favor of defining the needed macro in `ecolor`. As I said in the other comment, I should be able to do this over the weekend.